core/core.go

```diff
@@ -154,10 +162,28 @@ func NewTextile(ctx context.Context, conf Config) (*Textile, error) {
 		return nil, err
 	}
 	if conf.AddrPowergateAPI != nil {
-		t.powc, err = powc.NewClient(conf.AddrPowergateAPI, grpc.WithInsecure(), grpc.WithPerRPCCredentials(powc.TokenAuth{}))
+		target, err := TCPAddrFromMultiAddr(conf.AddrPowergateAPI)
 		if err != nil {
 			return nil, err
 		}
+		t.powc, err = powc.NewClient(target)
+		if err != nil {
+			return nil, err
+		}
+		pgConn, err := grpc.Dial(target, grpc.WithInsecure())
+		if err != nil {
+			return nil, err
+		}
+		t.powStub = grpcdynamic.NewStub(pgConn)
+		ffsDesc, err := desc.LoadFileDescriptor("ffs/rpc/rpc.proto")
+		if err != nil {
+			return nil, err
+		}
+		ffsServiceDesc := ffsDesc.FindService("ffs.rpc.RPCService")
+		if ffsServiceDesc == nil {
+			return nil, fmt.Errorf("no ffs service description found")
+		}
+		t.ffsServiceDesc = ffsServiceDesc
```

Comment:
We create the dynamic FFS service description by loading the file descriptor data from the runtime (`ffs/rpc/rpc.proto` is not a path to a file, but a key that the ffs file descriptor is registered under), then looking up the service we want (`ffs.rpc.RPCService`)